src/psyclone/tests/domain/common/transformations/alg_invoke_2_psy_call_trans_test.py

```diff
@@ -588,3 +589,42 @@ def test_ai2psycall_remove_imported_symbols(fortran_reader):
     trans.apply(invokes[1])
     assert "kern" not in module.symbol_table._symbols
     assert "kern_mod" not in module.symbol_table._symbols
+
+
+def test_ai2psycall_keep_comments():
+    '''Check that the apply() method works as expected when the invoke has
+    a single kernel with multiple fields of the same name. Also check
+    that the apply() method creates the required routine and container
+    symbols if they have not already been created. Also check that the
+    apply() method removes the invoke symbol from the appropriate
```

Comment:
I can't see that the test checks for the "invoke" symbol? I suggest that all this test needs to do is to check that comments are retained and reproduced in the output.

Reply:
Updated the docstring - this was a c/p job it looks like that I missed changing.